<launch>
    <node pkg="path_smoothing" type="path_smoothing_node" name="path_smoothing_node" args="" output="screen" >
         <rosparam command="load" file="$(find path_smoothing)/config/config.yaml" />
    </node>

    <node name="rviz" pkg="rviz" type="rviz" args="-d $(find path_smoothing)/rviz/path_smooth.rviz" output="screen" />

    <node pkg="rqt_reconfigure"  type="rqt_reconfigure" name="reconfigure_gui" />
</launch>
